identity/identity/services/health.py

```diff
@@ -28,12 +27,7 @@ def __init__(
             raise Exception()
 
     async def healthy(self) -> bool:
-        with concurrent.futures.ProcessPoolExecutor() as executor:
-            futures = [
-                executor.submit(self.user_repository.count),
-                executor.submit(self.token_cache.count),
-            ]
-
-        concurrent.futures.wait(futures)
```

Comment:
I reworked this to check exceptions and found out the calls weren't even happening due to pickling issues, so we'll make them serially for now.